themes/scalereal/static/css/style.css

```diff
@@ -3363,4 +3363,148 @@ lottie-player {
 
 .feedback-mobile  .owl-carousel .owl-dots {
   margin-top: -2px;
+}
+
+/* Client Success Stories Section */
+.client-success-stories-section .container .box {
+  padding: auto 98px ; 

```

Comment:
It's invalid, you can not use auto with pixel. and this is not at all applying on the   reactangle. Actual padding is coming from .box class which is 100px 70px. So even if you completely remove this then it wont affect your design